### ISSUE TITLE
Data tab not working

### ISSUE THREAD

**Issue by Riddlez** [emberjs/ember-inspector#739]
When in my ember project I am trying to select the data tab in the ember inspector but it not showing the data tab basically it does nothing

**Comment by teddyzeenny** [comment]
Hi, what are the Ember and Ember Data versions? Do you see any errors in the console after clicking the Data tab?

**Comment by Riddlez** [comment]
HI
I have no errors in my console
and am using the below versions in package json: 
"broccoli-asset-rev": "^2.4.5",
    "ember-ajax": "^3.0.0",
    "ember-cli": "~2.16.2",
    "ember-cli-app-version": "^3.0.0",
    "ember-cli-babel": "^6.6.0",
    "ember-cli-dependency-checker": "^2.0.0",
    "ember-cli-eslint": "^4.0.0",
    "ember-cli-htmlbars": "^2.0.1",
    "ember-cli-htmlbars-inline-precompile": "^1.0.0",
    "ember-cli-inject-live-reload": "^1.4.1",
    "ember-cli-mirage": "^0.4.0",
    "ember-cli-qunit": "^4.0.0",
    "ember-cli-sass": "^7.0.0",
    "ember-cli-shims": "^1.1.0",
    "ember-cli-sri": "^2.1.0",
    "ember-cli-uglify": "^2.0.0",
    "ember-data": "~2.16.2",
    "ember-export-application-global": "^2.0.0",
    "ember-load-initializers": "^1.0.0",
    "ember-resolver": "^4.0.0",
    "ember-simple-auth": "^1.4.0",
    "ember-simple-auth-token": "^2.1.0",
    "ember-source": "~2.16.0",
    "ember-toastr": "^1.7.0",
    "ember-truth-helpers": "^2.0.0",
    "ember-welcome-page": "^3.0.0",
    "loader.js": "^4.2.3",
    "toastr": "^2.1.2"

**Comment by teddyzeenny** [comment]
Can you point me to an online repo or app that I can access that reproduces this issue?

**Comment by lifeart** [comment]
@teddyzeenny `linkedin` ember-inspector data tab freezing

**Comment by vvscode** [comment]
@lifeart  it works for me http://take.ms/2gCit4

**Comment by Benjy1979** [comment]
I had the same issue, it's when the Web Inspector crashes. For instance in this case since 3,.o0 I had to change a model from:
`import Model from 'ember-data/model';
// import attr from 'ember-data/attr';
// import { belongsTo, hasMany } from 'ember-data/relationships';

export default Model.extend({
  user_id: DS.attr('number'),
  created: DS.attr(),
  caption: DS.attr(),
  plandata: DS.attr()
});
`
to 
```
import DS from 'ember-data';
// import attr from 'ember-data/attr';
// import { belongsTo, hasMany } from 'ember-data/relationships';

export default DS.Model.extend({
  user_id: DS.attr('number'),
  created: DS.attr(),
  caption: DS.attr(),
  plandata: DS.attr()
});
```
The web inspector crashed on the previous saying that DS didn't exist. It could have been an old model that I had defined myself awhile back which is no longer working. Or maybe old CLI behaviour, I can't recall, but this fixed it for me.

**Comment by teddyzeenny** [comment]
@Benjy1979 in the first example you're using `DS` without importing it, that's probably why it's crashing.

**Comment by Benjy1979** [comment]
Yes I agree, but it wasn't crashing in the previous version of Ember.js

**Comment by locks** [comment]
> Yes I agree it's a bug, but it wasn't crashing in the previous version of Ember.js.

Because up until 3.x, `DS` and `Ember` were available as globals.

**Comment by GottZ** [comment]
i +1 this issue. at least i can still navigate through models using the Routes tab but still.. kinda painful and rendering the feature i use the most kinda useless.

**Comment by rwwagner90** [comment]
@GottZ are you using `DS` without importing it?

**Comment by GottZ** [comment]
nope. `import DS from 'ember-data` everywhere. i just checked.
well.. i do inject a overridden store into the app and made a few overrides in my application adapter since our api is crap af and does not follow the jsonapi spec correctly. but besides that, nothing.

**Comment by rwwagner90** [comment]
@GottZ so this issue is resolved for you? @Riddlez are you still having issues?

**Comment by GottZ** [comment]
yup totally resolved for me. i also just upgraded to ember 3.0.0 without a problem. no issues with ember-inspector.

**Comment by omairrazam** [comment]
i faced the same issue with latest Ember

ember-cli: 3.21.2
node: 10.18.1
